charms/cert-manager-webhook/reactive/webhook.py

```diff
@@ -57,7 +57,11 @@ def start_charm():
     for _ in range(30):
         try:
             secret = v1.read_namespaced_secret(name="cert-manager-webhook-tls", namespace=namespace)
-            break
+            if secret.data.get('tls.crt'):
```

Comment:
@knkski i'm not familiar with `CoreV1Api().read_namespaced_secret()`. Assuming it doesn't throw an exception, will it always return something in `.data`, or do you need to check if `secret.data` is valid before you go `.get()` stuff from it?

Reply:
`secret.data` is part of the `Secret` resource, so it should always exist.